### PR TITLE
Timesync master & client bqs

### PR DESCRIPTION
**Pull request by jaz-jlh**

### CODE REVIEW

File: infrastructure/time/timesync_bq/client_timesync_message.cc [hunk 1]
```diff
@@ -0,0 +1 @@
+#include "infrastructure/time/timesync_bq/client_timesync_message.hh"
```
Comment (jpanikulam) [inline infrastructure/time/timesync_bq/client_timesync_message.cc]: No need for this file

File: infrastructure/time/timesync_bq/timesync_client_bq.cc [hunk 2]
```diff
@@ -0,0 +1,39 @@
+//%bin(timesync_client_bq)
```
Comment (jpanikulam) [inline infrastructure/time/timesync_bq/timesync_client_bq.cc]: In keeping with the other bq’s, ought to call this _main

**Comment by jpanikulam** [comment]
@jaz-jlh Did you get a chance to test this?

**Comment by jaz-jlh** [comment]
Not tested yet, planning to test today with @benkroop

**Comment by jpanikulam** [comment]
@jaz-jlh Please also include instructions about how to make use of this. How do you run it, and what is the effect?

Is the actual system time synchronized? Does it repeatedly synchronize? What happens if it's started as one of the later bq's?

**Comment by jaz-jlh** [comment]
tested locally

**Pull request merged**